notebooks/Fine_tuning_Code_LLM_on_single_GPU.ipynb

```diff
@@ -0,0 +1,5363 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def chars_token_ratio(dataset, tokenizer, data_column, nb_examples=400):
</pre><p>What does <code>nb_examples</code> stand for? Why is it set to 400? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Added a note.